airmozilla/popcorn/views.py

```diff
@@ -113,6 +113,31 @@ def popcorn_data(request):
         }
 
 
+@json_view
+@login_required
+@require_POST
+def save_edit(request):
+    slug = request.POST.get('slug')
+    if not slug:
+        return http.HttpResponseBadRequest('slug')
+    data = request.POST.get('data')
+    if not data:
+        return http.HttpResponseBadRequest('data')
+
```

Comment:
I think we should check here that this string can be converted from a JSON string to a python dict object. 
And if it's empty (e.g. `{}`) we should not accept it.